qcelemental/models/molecule.py

```diff
@@ -177,8 +194,7 @@ def must_be_n_frag(cls, v, values, **kwargs):
                 raise ValueError("Fragment Charges and Fragment Multiplicities"
                                  " must be same number of entries as Fragments")
         else:
-            raise ValueError("Cannot have Fragment Charges or Fragment Multiplicities "
-                             "without Fragments")
+            raise ValueError("Cannot have Fragment Charges or Fragment Multiplicities " "without Fragments")
```

Comment:
pertains to `populate_real` [above](https://github.com/MolSSI/QCElemental/pull/20/files#diff-c1df0b88317a0c0331bea59a1793059cR187): so once qcschema_molecule exists, field `real` guaranteed present? this may change intf_dftd3 a smidge

Reply:
sounds good. in dftd3 I already have to form the molrec for `to_string`, so i'll just keep copying `'real'` until dust settles.